src/utils.js

```diff
@@ -1,8 +1,11 @@
-const Chance = require('chance');
+const Chance = require("chance");
```

Comment:
Oops sorry, I'm using [prettier](https://github.com/prettier/prettier) in my editor and it has formatted the whole file. I can revert all unrelated changes if you want.